src/components/widget/templates/mangrove-alerts/component.js

```diff
@@ -49,11 +49,57 @@ function MangroveAlerts({ widgetConfig, minDate = '1996-01-01', maxDate = '2016-
     onChange={value => datepickerHandler({endDate: value})}
   />);
 
-  const startYear = year(state.startDate);
-  const endYear = year(state.endDate);
+  const startMark = month(state.startDate) + 2;
+  const endMark = month(state.endDate) - 2;
 
-  const { chartData, chartConfig } = widgetConfig.parse({ startYear, endYear });
-  const alerts = chartData.map(it => it.year).filter(y => y >= startYear && y <= endYear ).length;
+  const seriesLineDefinition = {
+    stroke: 'red',
+    strokeWidth: 1,
+    isAnimationActive: false,
+    dot: {
+      fill: 'red',
+      stroke: 'red',
```

Comment:
According to design this color should be #EB6240
```suggestion
      stroke: '#EB6240',
```